### PR TITLE
Exit with code 1 when validation fails

### PR DESCRIPTION
**Pull request by smcveigh941**
## Environment
**Liquibase Version**: 4.1.1

**Liquibase Integration & Version**: CLI

## Pull Request Type

* [x] Bug fix (non-breaking change which fixes an issue.)
* [ ] Enhancement/New feature (non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Description
A clear and concise description of the issue being addressed.  Additional guidance [here](https://liquibase.jira.com/wiki/spaces/LB/pages/1274904896/How+to+Contribute+Code+to+Liquibase+Core).

* Checksum error on Validate exits with code 0, should be non-zero
* See issue #1006 

## Steps To Reproduce
* Create a checksum error by altering a changelog file after migrating to a database
* Run `liquibase validate` with the appropriate parameters

## Actual Behavior
* Liquibase commandline exits with code 0 (i.e. everything is successful)
image

## Expected/Desired Behavior
* Liquibase commandline should exit with non-zero exit code

## Fast Track PR Acceptance Checklist:
* [x] Build is successful and all new and existing tests pass
* [ ] Added [Unit Test(s)](https://liquibase.jira.com/wiki/spaces/LB/pages/1274937609/How+to+Write+Liquibase+Core+Unit+Tests)
* [ ] Added [Integration Test(s)](https://liquibase.jira.com/wiki/spaces/LB/pages/1276608569/How+to+Write+Liquibase+Core+Integration+Tests)
* [x] Documentation Updated

### CODE REVIEW

**Comment by smcveigh941** [comment]
Hi @SteveDonie, this PR is to resolve issue #1006 

Can you invite me to the discord so we can have a quick chat about this please?

**Comment by smcveigh941** [comment]
@ro-rah @molivasdat Hey guys, I am satisfied with this and tests are passing. Having issues increasing code coverage because it seems this area of code isnt very testable without a refactor... find my evidence of the fix below :)

image

**Comment by smcveigh941** [comment]
@molivasdat gentle reminder of this :) thanks

File: liquibase-core/src/main/java/liquibase/integration/commandline/Main.java [hunk 1]
```diff
@@ -155,7 +155,7 @@ public static void main(String[] args) {
         try {
             errorLevel = run(args);
         } catch (Throwable e) {
-            System.exit(-1);
+            System.exit(1);
```
Comment (molivasdat) [inline liquibase-core/src/main/java/liquibase/integration/commandline/Main.java:158]: Changing from -1 to positive 1 is still a non-zero return code.  So this one does not look necessary.

**Comment by molivasdat** [comment]
Hi @smcveigh941.  Thanks for the reminder. I made one comment.  Checking with development on the rest.

**Comment by nvoxland** [comment]
We've made enough changes to the CLI framework, that I'm going to close this PR in favor of #2185 which takes the parts we need now without changing around things that no longer need to be.

**Pull request closed without merging**